seed/seed.json

```diff
@@ -2653,6 +2653,37 @@
                     "ANDROID"
                 ]
             }
+        },
+        {
+            "name": "BraveAdblockExperimentalListDefaultStudy",
+            "experiments": [
+                {
+                    "name": "Enabled",
+                    "probability_weight": 100,
+                    "feature_association": {
+                        "enable_feature": [
+                            "BraveAdblockExperimentalListDefault"
+                        ]
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "channel": [
+                    "BETA",
+                    "NIGHTLY"
+                ],
+                "platform": [
+                    "WINDOWS",
+                    "MAC",
+                    "LINUX",
+                    "ANDROID"
+                ],
+                "min_version": "123.1.66.53"
```

Comment:
Because the [PR](https://github.com/brave/brave-core/pull/22858) will go into 1.66.53.

Reply:
I've marked it as going into Beta even though it's not in Beta yet but my understanding/hope is that `min_version` will take care of that.